apps/src/sites/studio/pages/projects/index.js

```diff
@@ -43,11 +46,18 @@ $(document).ready(() => {
         isRtl={isRtl}
         canViewFullList={true}
         canViewAdvancedTools={projectsData.canViewAdvancedTools}
+        shouldLogEvents={true}
       />,
       document.getElementById('new-project-buttons')
     );
   } else {
-    $('#legacy-project-links').show();
+    const legacyProjectLinks = $('#legacy-project-links');
+    legacyProjectLinks.show();
+    LEGACY_PROJECT_BUTTON_TYPES.forEach(projectType => {
+      legacyProjectLinks.find(`a[data-projectType=${projectType}]`)
+        .on('click', recordLegacyProjectButtonClick.bind(this, projectType));
```

Comment:
This might be a dumb question: Could I throw the stats by clicking on this button a whole bunch before the page successfully navigates?  I wonder if we should somehow debounce this or enforce that it only fires once per page load.

Reply:
you could, however it's also possible that someone would command-click (middle click, etc) to open many of these projects in new windows, so we wouldn't want to limit this to 1. debounce seems appropriate. I've added one with a 1s delay.